
<launch>
  <node pkg="location_match_score" type="location_match_score" name="location_match_score" output="screen">
    <!-- <remap from="scan" to="scan"/> -->
    <param name="odom_frame" value="odom"/>
     <param name="throttle_scans" value="1"/>
    <param name="use_map_topic" value="true"/>
    <param name="correlation_search_space_resolution" value="0.01"/>
    <param name="correlation_search_space_dimension" value="0.2"/>
    <param name="correlation_search_space_smear_deviation" value="0.05"/>
  </node>
</launch>



